doc/user/quick-sql-grammar.md

```diff
@@ -228,6 +229,15 @@ Generate PL/SQL APIs on all tables for create, insert, update, delete and query.
 Adds an additional created, created_by, updated and updated_by columns to every
 table created.
 
+### boolean
+
+**Possible Values**: `yn`, `native`  
+**Default Value**: `false`
+
+Adds an additional created, created_by, updated and updated_by columns to every
+table created.
```

Comment:
This text does not make sense for boolean, boolean is for the datatype boolean column, not for audit columns. I I think this was a copy and paste. Please update the text accordingly. But I am questioning if this setting should exist at all.

I thought we were piggybacking off of `db` setting. If db = 23c, then treat boolean differently.

If I have db = 19c, and I set this setting -> `native` then we are in a weird spot where the db version doesn't really mean anything for us.

Can we remove this setting externally? It is ok to use it internally by reading the `db` setting, and determining what to do.

Reply:
@vadim-tropashko Can you please let me know about this setting and if we can remove it as it can go against the `db` setting?

Reply:
Also the text in it still is talking about audit columns, but again would live it removed completely

Reply:
Here is what is currently there:

`**Possible Values**: `yn`, `native`  
**Default Value**: inferred from #db

Set boolean to legacy char(1) or new 23c native boolean value. This setting has priority
over db:23c, so that the user can override db seting (which influences other functionality)
`